python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -283,6 +345,12 @@ def get_snfluxes(self):
                 vals[:, 16], vals[:, 17], vals[:, 18])
 
 
+    def get_EBV(self):
```

Comment:
What you have does work, it just seems redundant with code we already have.

The `get_EBV` method on `EBVmixin` bases its E(B-V) calculation on the galactic longitude (`glon`) and galactic latitude (`glat`) of the object.  As long as you provide a way to calculate those quantities for the supernovae, you should be able to use it.

One warning: `EBVmixin` also contains a method `get_galacticAv`, which calculates Av as `EBV*galacticRv`.  That will override anything that is in the database you're connecting to.  I don't know if that is a problem for supernovae or not.